### PR TITLE
docs/readme: Add Bugzilla issue reporting link

### PR DESCRIPTION
**Pull request by ironcladlou**
Bugs are tracked in Bugzilla. GitHub issues are disabled for the repository, so make a note in the README to redirect users to Bugzilla.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,7 @@ Cluster DNS Operator manages CoreDNS as a Kubernetes DaemonSet exposed as a Serv
 ## How to help
 
 See [HACKING.md](HACKING.md) for development topics.
+
+## Reporting issues
+
+Bugs are tracked in [Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform).
```
Comment (Miciah) [inline README.md]: Preselect component and version? https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform&version=4.0.0&component=Networking

Comment (ironcladlou) [reply README.md]: Nice, didn't know you could do that

**Comment by ironcladlou** [comment]
I wanna squash

/lgtm cancel

**Comment by Miciah** [comment]
Should we use the "Routing" component or "Networking" for DNS? (Or is there another option?)

**Comment by ironcladlou** [comment]
"Routing" maps to our team, for now.

**Comment by ironcladlou** [comment]
@Miciah squashed, PTAL

**Comment by Miciah** [comment]
All right,
/lgtm

**Pull request merged**